onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -3732,16 +3732,40 @@ def aten_logaddexp2(self: TFloatOrBFloat16, other: TFloatOrBFloat16) -> TFloatOr
     return op.Div(op.Log(summation), op.Log(2.0))
 
 
-@torch_op("aten::logcumsumexp")
-def aten_logcumsumexp(self: TFloatOrBFloat16, dim: INT64) -> TFloatOrBFloat16:
+@torch_op("aten::logcumsumexp", trace_only=True)
+def aten_logcumsumexp(self: TFloatOrBFloat16, dim: int) -> TFloatOrBFloat16:
     """logcumsumexp(Tensor self, int dim) -> Tensor"""
 
-    if op.Size(op.Shape(self)) == 0:
+    return _aten_logcumsumexp_onnx(self, [dim])
+
+@torch_op("aten::logcumsumexp", private=True)
+def _aten_logcumsumexp_onnx(self: TFloatOrBFloat16, dims: Sequence[int]) -> TFloatOrBFloat16:
```

Comment:
I am not sure what it means to have a sequence of axes as dims. Since the operation involves a prefix-scan / cumulative-sum, not clear how to generalize from 1-dimension to 2-dimensions.

Reply:
Changed